src/components/radioFieldSet.js

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types';
+import {
+  Button, Fieldset,
+  MultiChoice, Radio
+} from 'govuk-react'
+import { Field, Form } from 'react-final-form';
+
+
+const required = (value) => (value ? undefined : 'Required');
+
+const RadioGroup = ({label, hint, options, input, meta}) => (
+  <div>
+    <MultiChoice label={label} hint={hint} meta={meta}>
+      {options.map((o) => (
+        <div key={o.value}>
+          <Radio {...input} value={o.value} checked={o.value === input.value}>
```

Comment:
Is checked={o.value === input.value} meant to make the radio button checked if that option was selected previously?

If so, it's not behaving correctly, i.e. select an option, click continue, click back and that option is not shown as selected.
This isn't a new bug, it exists in the main deployed version too - wanted to know if the above line is meant to handle the case?